samples/apps/copilot-chat-app/webapp/src/libs/services/GraphService.ts

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import { BaseService } from './BaseService';
+
+export interface BatchRequest {
+    id: string;
+    method: 'GET' | 'POST' | 'PUT' | 'UPDATE' | 'DELETE';
+    url: string;
+    headers: any;
+}
+
+export interface BatchResponseBody {
+    responses: BatchResponse[];
+}
+
+export interface BatchResponse {
+    id: number;
+    status: number;
+    body?: any;
+    headers?: any;
+}
+
+export class GraphService extends BaseService {
+    constructor() {
+        super('https://graph.microsoft.com');
+    }
+
+    private version = 'v1.0';
+
+    private getCommandPath = (resourcePath: string) => {
+        return `/${this.version}/${resourcePath}`;
+    };
+
+    public makeBatchRequest = async (batchRequests: BatchRequest[], accessToken: string) => {
+        const result = await this.getResponseAsync<BatchResponseBody>(
+            {
+                commandPath: this.getCommandPath('$batch'),
+                method: 'POST',
+                body: { requests: batchRequests },
+            },
+            accessToken,
+        );
+
+        return result.responses;
+    };
+
+    public loadUser = async (userId: string, accessToken: string) => {
+        const result = await this.getResponseAsync<object>(
+            {
+                commandPath: this.getCommandPath(`users/${userId}`),
+                method: 'GET',
+            },
+            accessToken,
+        );
+
+        return result;
+    };
```

Comment:
Is this used anywhere?

Reply:
No, I thought it might be helpful for future work when we load user info for each message, but I'll remove until we actually need it